app/models/petition.rb

```diff
@@ -193,6 +189,10 @@ def closed?
     self.state == OPEN_STATE && self.closed_at <= Time.current
   end
 
+  def in_todo_list?
+    state == SPONSORED_STATE || state == VALIDATED_STATE || state == PENDING_STATE
```

Comment:
Nitpick: we could simplify this to: `[SPONSORED_STATE, VALIDATED_STATE, PENDING_STATE].include? state`